### DISCUSSION TITLE
Direction of display for devices

### DISCUSSION THREAD

**Discussion by rhuebner76** [usdot-jpo-ode/jpo-wzdx#248] [Q&A]
With the inclusion of devices on the roadway properties of location have been reduced to a single milepost and a map point. However, there is no property indicating which direction of travel the device will be seen on or pertains to.

**Comment by j-d-b** [comment]
This is a great comment and something we are planning on addressing in the v4.1 release through adding a `direction` or `road_direction` property to the [FieldDeviceCoreDetails](https://github.com/usdot-jpo-ode/wzdx/blob/main/spec-content/objects/FieldDeviceCoreDetails.md). I'm going to move this to [Issues](https://github.com/usdot-jpo-ode/wzdx/issues).

**Reply by j-d-b** [reply]
I created the issue: see #250.
